readme.md

```diff
@@ -61,3 +61,5 @@ If `true`, the resulting ZIP file contents will be a buffer. Large zip files may
 If `false`, the ZIP file contents will be a stream.
 
 Similar to [gulp.src's `buffer` option](https://gulpjs.com/docs/en/api/src/#options).
+
+Tip: set this to `false` to allow creating ZIP files larger than Node's maximum buffer length.
```

Comment:
This is just repeating what's on line 60. I think the option docs should explain why it exists when there's already such option for `gulp.src`.

Reply:
Would something like this work:

> We use this option instead of relying on `gulp.src`'s `buffer` option because we are mapping many input files to one output file and can't reliably detect what the output mode should be based on the inputs, since Vinyl streams could contain mixed streaming and buffered content.

Reply:
Yes, that's good.